docs/content/docs/operators/feature/vectorassembler.md

```diff
@@ -26,10 +26,20 @@ under the License.
 -->
 
 ## Vector Assembler
-
-Vector Assembler combines a given list of input columns into a vector column.
-Types of input columns must be either vector or numerical value.
-
+A Transformer which combines a given list of input columns into a vector column. Input columns
+would be numerics or vectors whose size is specified by the {@link #INPUT_SIZES} parameter.
```

Comment:
nit: `Input columns would be numerics` --> `Input Columns could be numerical or vectors whose sizes are specified by...`

Same for the java docs in python and java source code.